index.html

```diff
@@ -7,8 +7,6 @@
   <title>Ugboaja Uchechi's Portfolio</title>
```

Comment:
[  OPTIONAL ] Please add unnecessary files and folders like `.vscode` to .gitignore.